packages/preview/clean-hda/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "clean-hda"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Roland Schätzle <@roland-KA>", "Stefan Siber <@stefan-ctrl>"]
+license = "MIT"
+description = "A Typst Template for Hochschule Darmstadt (h_da) - University of Applied Sciences"
```

Comment:
The description should avoid starting with `A template` or the words `typst`, because they are kind of redundant. See [this](https://github.com/typst/packages/blob/main/docs/manifest.md#writing-a-good-description) for additional guidance.